build/README.md

```diff
@@ -90,20 +90,20 @@ export KUBE_RSYNC_PORT=8730
 docker-machine ssh ${KUBE_BUILD_VM} -L ${KUBE_RSYNC_PORT}:localhost:${KUBE_RSYNC_PORT} -N &
 ```
 
-Look at `docker-machine stop`, `docker-machine start` and `docker-machine rm` to manage this VM.
+Look at `docker-machine stop`, `docker-machine start`, and `docker-machine rm` to manage this VM.
```

Comment:
this isn't necessary

Reply:
We decided long ago that Kubernetes respects the Oxford comma.  I wouldn't send a PR just for this, though.

Reply:
Do we have an oxford comma linter yet?